src/hearings/converters/date-response-submitted.answer.converter.ts

```diff
@@ -9,8 +9,8 @@ export class DateResponseSubmittedAnswerConverter implements AnswerConverter {
   public transformAnswer(hearingState$: Observable<State>): Observable<string> {
     return hearingState$.pipe(
       map(state => {
-        const hearingResponseTime = state.hearingRequest.hearingRequestMainModel.hearingResponse.hearingDaySchedule[0].hearingStartDateTime;
-        return moment(hearingResponseTime).format(HearingDateEnum.DisplayMonth);
+        const hearingStartDateTime = state.hearingRequest.hearingRequestMainModel.hearingResponse.hearingDaySchedule[0].hearingStartDateTime;
```

Comment:
do we have to check if the hearingDaySchedule length is valid before we take the [0] array?

Reply:
fair point now add the length check